src/frontend/src/lib/utils/transaction.utils.ts

```diff
@@ -69,7 +70,7 @@ export const groupTransactionsByDate = <T extends AnyTransactionUi>(
 	}, {});
 };
 
-export const mapTransactionModalData = <T extends AnyTransactionUi>({
+export const mapTransactionModalData = <T extends AnyTransactionUi | Transaction>({
```

Comment:
Doesn't `AnyTransactionUi` extends `Transaction`?

`<T extends Transaction>`?

Reply:
No indeed, only `EthTransactionUi`, but the Ethereum token-specific transaction list uses type `Transaction`, I will analyze the reason why